docs/first-step/run-executable/cli-xcode.rst

```diff
@@ -4,15 +4,15 @@
 CLI: Xcode
 ----------
 
-To build Xcode project from command line ``xcodebuild`` can be used.
+To build and Xcode project from the command line, ``xcodebuild`` can be used.
```

Comment:
`To build and Xcode` -> `To build an Xcode`

Reply:
yup, thanks